src/core/worker.js

```diff
@@ -436,6 +437,44 @@ var WorkerMessageHandler = PDFJS.WorkerMessageHandler = {
       });
     });
 
+    handler.on('GetRawObject', function wphGetRawObject(data) {
+      function clone(obj) {
+        if (!obj || isBool(obj) || isNum(obj) || isString(obj)) {
+          return obj;
+        } else if (isName(obj)) {
+          return { name: obj.name };
+        } else if (isDict(obj) || isArray(obj)) {
+          return obj.map(clone);
```

Comment:
Another reviewer asked for some changes from the original PR. Can you give me more detail about what you think is wrong here?